book/website/collaboration/inclusive-events/inclusive-events-schedules.md

```diff
@@ -0,0 +1,286 @@
+For example, Day 1 of a 3-Day event might schedule the majority of sessions focused on public engagement and lived experience, followed by Day 2 which could focus on technical workshops and other content. 
+Alternatively, if your conference is online, consider choosing a repeating session time from 10 a.m. - 1 p.m. on all 3 days that could focus on lived experience content. 
+
+This can support the engagement of people with lived experience who volunteer their time to engage in research, as well as people with lived experience of disabilities and medication conditions that can significantly impact their energy levels. 
+
+When deciding where in your schedule to plan content, ensure that you work with, or ask people with lived experience from your specific audience what would works for them, instead of assuming and choosing times that are otherwise empty in the schedule.
+
+(cl-inclusive-events-schedule-content)=
+## Event Content: Planning for Engagement
+When identifying the content of your event, you should spend time identifying what type of content and sessions attendees with lived experience would be interested in, what "engagement" means for your specific event, and how will you ensure that attendees with lived experience are able to engage in your event.
+
+By clearly defining what engagement looks like, and also how to engage, your event will be more accessible to people with lived experience and also reduce potential barriers and anxieties for participants.
+
+(cl-inclusive-events-defining-engagement)=
+### Defining Engagement for Your Event
+Providing guidance on what engagement means for your specific event can help to set expectations, and also reduce potential barriers and anxieties that attendees with lived experience may have when trying to engage in your event.  
+For example, people with an experience of cognitive impairment, as well as autistic people, can find unstructured sessions daunting which will negatively impact their ability to fully engage. 
+
+The guidance you provide will be different depending on your event. 
+For example, a public lecture may ask attendees to submit questions for the speaker at the end of the lecture. 
+For this example, indicate in your guidance how long there will be for questions, how to use any online software you may be using to collect audience questions, and also how many questions you are likely going to answer. 
+This can help avoid any feeling attendees may have about being ignored or not heard if their question is not addressed. 
+
+For other events that require active participation, such as a focus group or discussions, consider including guidance on the format discussions will have, and if there are any specific topics or prior material attendees should be familiar with in order to take part. 
+Try to identify if your event will facilitate discussion between attendees, or if the sessions are to be chaired and led by a facilitator. 
```

Comment:
Based on this comment (and your insight) I think we'll add a section to the chapter which covers "guide for facilitators and moderators" and will build out that idea bit more 🚀 


I'll leave this comment unresolved for now so that we can cross-reference that new section once it's drafted in here.